### PR TITLE
Make DIM work without gmake

### PR DESCRIPTION
**Pull request by dberzano**
@matthiasrichter this obsoletes #973 as discussed

### CODE REVIEW

**Comment by dberzano** [comment]
@Barthelemy any objection?

File: dim.sh [hunk 1]
```diff
@@ -20,6 +20,11 @@ cd $FILE_NAME
 tr -d '\015' < setup.sh > setup2.sh
 mv setup2.sh setup.sh
 
+# DIM installation needs `gmake` (`make` is not sufficient)
```
Comment (ktf) [inline dim.sh]: Shouldn't we patch DIM and propose the patch upstream?

Comment (Barthelemy) [reply dim.sh]: We could contact Clara Gaspar indeed. The project is in maintenance but they take patches.

**Comment by Barthelemy** [comment]
It was any ways working on mac and CC7 until now as far as I could test it. The change is ok and does not break anything.

File: dim.sh (hunk 1 above)
Comment (dberzano) [reply dim.sh]: That was the original proposal discussed with @matthiasrichter, but we don't know the reaction time and it breaks the recipe for old releases (given that the oldest release probably dates back to years ago...) I actually propose we do both.

**Comment by dberzano** [comment]
@Barthelemy it worked on systems where `gmake -> make`. There are corner cases where the symlink is not available (such as @matthiasrichter's system).

File: dim.sh (hunk 1 above)
Comment (Barthelemy) [reply dim.sh]: I think that it makes sense to do both

**Comment by matthiasrichter** [comment]
exactly, I'm working on Ubuntu, `make` is actually `gmake` on such systems because its GNU. And there is no extra gmake package at all.

**Pull request merged**